serving/docker/scripts/install_docker.sh

```diff
@@ -7,4 +7,5 @@ fi
 
 curl -fsSL https://get.docker.com -o get-docker.sh
 sudo sh get-docker.sh
-sudo chmod 666 /var/run/docker.sock
+# grant docker permission
+sudo usermod -a -G docker ubuntu
```

Comment:
this wont' work, you need restart ssh session to activate group permission

Reply:
This will be helpful for AARCH JNI build. All future usage of docker would work